cmd/config/cnv-metrics.yml

```diff
@@ -0,0 +1,124 @@
+---
+# OpenShift virtualization
+# elapsed is the job duration time
+## metadata
+- query: kubevirt_hyperconverged_operator_health_status{}
+  metricName: hco-operator-health
+  instant: true
+
+- query: sum(kubevirt_number_of_vms)
+  metricName: total-vm-created
+  instant: true
+
+- query: sum(cnv:vmi_status_running:count)
+  metricName: total-vmi-running
+  instant: true
+
+- query: cluster:vmi_request_cpu_cores:sum
+  metricName: total-cpu-request
+  instant: true
+
+- query: sum(kubevirt_vm_resource_requests)
+  metricName: total-mem-request
+  instant: true
+
+- query: sum(up{namespace="openshift-cnv",pod=~"virt-api-.*"}) or vector(0)
```

Comment:
What's the meaning of this expression?

Reply:
This is getting the number of running virt-api pods, if none exists, returns 0.